testing/sdk_metrics.py

```diff
@@ -137,7 +137,7 @@ def get_metrics(package_name, service_name, pod_name, task_name):
         retry=False,
     )
     app_json = json.loads(app_response.text)
-    if app_json["dimensions"]["executor_id"] == task_to_check.executor_id:
+    if app_json["dimensions"]["executor_name"] == task_name:
```

Comment:
@kvish looking at what I'm seeing from the metrics in #2664, I don't think that `executor_name` exists in the metrics either.

Reply:
Based on a [diff](https://jira.mesosphere.com/browse/DCOS_OSS-4090?focusedCommentId=178332&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-178332) Kevin posted on JIRA it should be `"task_name"`:
```diff
diff --git a/frameworks/kafka/tests/test_sanity.py b/frameworks/kafka/tests/test_sanity.py
index e890fd643..069abea30 100644
--- a/frameworks/kafka/tests/test_sanity.py
+++ b/frameworks/kafka/tests/test_sanity.py
@@ -151,10 +151,10 @@ def test_pod_replace():
 @pytest.mark.sanity
 @pytest.mark.metrics
 @pytest.mark.dcos_min_version("1.9")
-@pytest.mark.skipif(
-    sdk_utils.dcos_version_at_least("1.12"),
-    reason="Metrics are not working on 1.12. Reenable once this is fixed",
-)
+#@pytest.mark.skipif(
+#    sdk_utils.dcos_version_at_least("1.12"),
+#    reason="Metrics are not working on 1.12. Reenable once this is fixed",
+#)
 def test_metrics():
     expected_metrics = [
         "kafka.network.RequestMetrics.ResponseQueueTimeMs.max",
diff --git a/testing/sdk_metrics.py b/testing/sdk_metrics.py
index 236d4299f..b7aad97eb 100644
--- a/testing/sdk_metrics.py
+++ b/testing/sdk_metrics.py
@@ -137,7 +137,7 @@ def get_metrics(package_name, service_name, pod_name, task_name):
         retry=False,
     )
     app_json = json.loads(app_response.text)
-    if app_json["dimensions"]["executor_id"] == task_to_check.executor_id:
+    if app_json["dimensions"]["task_name"] == task_name:
         return app_json["datapoints"]
 
     raise Exception("No metrics found for task {} in service {}".format(task_name, service_name))
```

Reply:
cool will make the change